sources/packages/web/src/components/fa-application/ConfirmStudentAidBCProfileInfo.vue

```diff
@@ -35,19 +35,18 @@
       <div class="p-field p-col-4">
         <label for="homeAddress" class="label-extra">Home Address</label>
         <div class="p-field p-col text-extra">
-          {{ studentConfirmInfo.addressLine1 }},<br />{{
-            studentConfirmInfo.addressLine2
-          }}<br />{{ studentConfirmInfo.city }}
-          {{ studentConfirmInfo.provinceState }}
-          {{ studentConfirmInfo.postalCode }}<br />{{
-            studentConfirmInfo.country
-          }}<br />
+          {{ studentContact.addressLine1 }},<br />
```

Comment:
I believe that this should be a computed property or something that would allow us to, for instance, do not display a blank line for address line 2 when it was not present. If I am not wrong this is the only non-required field.

Reply:
Actually not related to the user story directly, but have fixed it. If there are more formatting changes needed, can do that after this one is merged in a subsequent PR.